sample-code/src/orchestration.ts

```diff
@@ -305,7 +305,26 @@ export async function orchestrationCompletionMasking(): Promise<
       masking_providers: [
         buildDpiMaskingProvider({
           method: 'pseudonymization',
-          entities: ['profile-email', 'profile-person']
+          entities: [
+            'profile-email',
+            'profile-person',
+            {
+              type: 'custom',
+              regex: '\\b[0-9]{4}-[0-9]{4}-[0-9]{3,5}\\b',
+              replacement_strategy: {
+                method: 'constant',
+                value: 'REDACTED_ID'
+              }
+            },
```

Comment:
[pp] Can we have a standard entity here with replacement strategy set to `fabricated_data` to show-case all methods?